internal/command/meta_backend.go

```diff
@@ -94,6 +95,9 @@ type BackendWithRemoteTerraformVersion interface {
 func (m *Meta) Backend(ctx context.Context, opts *BackendOpts, enc encryption.StateEncryption) (backend.Enhanced, tfdiags.Diagnostics) {
 	var diags tfdiags.Diagnostics
 
+	_, span := tracing.Tracer().Start(ctx, "Initialize Backend")
```

Comment:
Why not overwriting the context? 🤔

Reply:
Ooops! looks like a mistake. I've fixed this now.